app/views/admin/workshop/_attendances.html.haml

```diff
@@ -6,7 +6,9 @@
           %td.small-1.columns
             - if (invitation.parent.date_and_time - 30.minutes).past?
               - if invitation.attended.nil?
-                = link_to admin_workshop_invitation_path(@workshop, invitation, attended: true), class: 'verify_attendance', title: 'Verify attendance', method: :put, remote: true do
+                = link_to admin_workshop_invitation_path(@workshop, invitation, attended: true),
+                  class: 'verify_attendance', title: 'Verify attendance', method: :put, remote: true,
+                  data: { disable_with: "<i class='fa fa-spinner spin'></i>" } do
```

Comment:
This `data-disable-with` trick is a nice things a lot of Rails app have, [rails-ujs (Unobtrusive JavaScript)](https://m.patrikonrails.com/a-definitive-guide-to-railss-unobtrusive-javascript-adapter-ef13bd047fff#dfbe).